### DISCUSSION TITLE
How to enable full screen when screen orientation changes?

### DISCUSSION THREAD

**Discussion by poudyalanil** [jhomlala/betterplayer#609] [Q&A]
Hi there, 
I am trying to enable full screen mode when screen orientation is changed to landscape from portrait.

``` 
if (MediaQuery.of(context).orientation == Orientation.landscape) {
            _betterPlayerController.enterFullScreen();
        }
```
this is what i have tried so far.

**Comment by SinoMiles** [comment]
BetterPlayerConfiguration(   autoDetectFullscreenDeviceOrientation: true）
if that doesn't work.you need may check your DeviceOrientation like following code
   
    SystemChrome.setPreferredOrientations([
      DeviceOrientation.landscapeLeft,
      DeviceOrientation.landscapeRight,
      DeviceOrientation.portraitUp,
    ]);

**Comment by jhomlala** [comment]
This has been solved already here: https://github.com/jhomlala/betterplayer/issues/117.

**Comment by dakontiva** [comment]
I am also trying to figure out how to enable fullscreen when the device orientation changes to landscape. I have looked at the example code in #117 but coudln't quite figure it out. So, I need to add an event listener to the controller and the handler should set the preferred orientations to landscape. Is that right?
